### ISSUE TITLE
Missing aria-disabled property on button components

### ISSUE THREAD

**Issue by jeffinnes** [lsst-epo/epo-react-lib#406]
## Context

In the [`OrbitalDetails` component](https://github.com/lsst-epo/epo-react-lib/blob/develop-v3/packages/epo-widget-lib/src/widgets/OrbitalSim/Orbitals/OrbitalDetails.jsx) we are passing a boolean to the `isInactive` prop on the `Styled.ButtonWrapper` that seems to be intended to set the `aria-disabled` value on the rendered button. 

## Issue 
The `aria-disabled` value of this prop is not being set on the "Show Details" button. It is possible that the issue lies with the underlying button component, but some discovery is required.
